packages/app/features/settings/screen.tsx

```diff
@@ -0,0 +1,26 @@
+import { View, StyleSheet, Text } from 'react-native'
+import React from 'react'
+
+export { SettingsScreen as default } from 'app/features/settings/screen'
+
+
+export function SettingsScreen() {
```

Comment:
I know that adding a `SettingsScreen` was the tutorial, but we don't need that.
What about removing it? 🙂

Reply:
Removed